tools/db_crashtest.py

```diff
@@ -56,7 +56,7 @@
     if random.randint(0, 1) == 0
     else random.choice(["none", "snappy", "zlib", "lz4", "lz4hc", "xpress", "zstd"]),
     "checksum_type": lambda: random.choice(
-        ["kCRC32c", "kxxHash", "kxxHash64", "kXXH3"]
```

Comment:
Switching this `enable_checksum_handoff` back off may be more relevant to these failures: https://github.com/facebook/rocksdb/blob/4e226c97b8328fdfccd2357ebdf3658b46806014/tools/db_crashtest.py#L265

Since the `checksum_type` has been here for some time, I think it's the recent `enable_checksum_handoff` to true that triggered this code path.